instrumentation-core/spring/spring-webmvc-3.1/README.md

```diff
@@ -0,0 +1,82 @@
+# Manual Instrumentation for Spring-WebMvc
```

Comment:
Spring Web MVC

We should copy from upstream docs / config for names as much as possible.

https://github.com/spring-projects/spring-framework/blob/master/spring-webmvc/spring-webmvc.gradle#L1